tests/test_hello_world.py

```diff
@@ -92,17 +112,60 @@ def test_make_me_2_functions_one_use_fixture_and_one_use_parametrize():
         assert times_7(rnd_int) == sum([rnd_int for i in range(7)])
 
         # assert times_7(rnd_int) > rnd_int  # TODO Explain why this assert doest work
+        #if rnd_int<0 then 7*rnd_int<0, then times_7(rnd_int) < rnd_int
 
 
 # TODO Add a function and at least 3 tests
+def power_2(a):
+    return a**2
+
+@pytest.mark.parametrize("a,b", [(2,4),(4,16),(0,0),(-1,1),(1.5,2.25)])
+def test_power_2_use_parametrize(a,b):
+    assert power_2(a) == b
+
+
+@pytest.fixture()
+def rnd_int_list():
+    random_generator = random.Random()
+    return [random_generator.randint(-1000, 1000) for i in range(10)]
+
+def test_power_2_use_fixture(rnd_int_list):
+    for rnd_int in rnd_int_list:
+        assert power_2(rnd_int) == rnd_int*rnd_int
+
+def test_power_2(rnd_int_list):
```

Comment:
Better function name
```suggestion
def test_power_2_always_greater_than_0(rnd_int_list):
```